src/PkgDeps.jl

```diff
@@ -173,10 +123,11 @@ end
 """
     users(pkg_name::String, pkg_registry::RegistryInstance; kwargs...)
 
-Find the users of the package named `pkg_name` which is registered in `pkg_registry`.
+Find the users of `pkg_name` which is registered in `pkg_registry_name`.
+Optionally limit search for users of `pkg_name` in specific registries by passing in kwarg `registries`.
 """
-function users(pkg_name::String, pkg_registry::RegistryInstance; kwargs...)
-    uuid = _get_pkg_uuid(pkg_name, pkg_registry)
+function users(pkg_name::String, pkg_registry_name::RegistryInstance; kwargs...)
+    uuid = _get_pkg_uuid(pkg_name, pkg_registry_name)
```

Comment:
why change the variable name here? this isn't just the registry_name, this is the registry.
```suggestion
function users(pkg_name::String, pkg_registry::RegistryInstance; kwargs...)
    uuid = _get_pkg_uuid(pkg_name, pkg_registry)
```